addons/l10n_tr/models/account_chart_template.py

```diff
@@ -0,0 +1,21 @@
+from odoo import models
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = "account.chart.template"
+
+    def _load(self, sale_tax_rate, purchase_tax_rate, company):
+        res = super()._load(sale_tax_rate, purchase_tax_rate, company)
+        if company.account_fiscal_country_id.code == 'TR':
+            company.account_sale_tax_id = self.env.ref(f"l10n_tr.{company.id}_tr_kdv_sale_18")
+            company.account_purchase_tax_id = self.env.ref(f"l10n_tr.{company.id}_tr_kdv_purchase_18")
```

Comment:
@qdp-odoo I could be interesting to have these fields be fields on the chart template. What do you think?